README.md

```diff
@@ -18,12 +18,14 @@ If the optional parameter relation-id is provided then only the information of t
 
 The following action will return all certificate requests that don't have certificates already provided, along with further information (relation-id, application_name and unit_name)
 
+NOTE: If you happen to scale `manual-tls-certificates`, you must run actions on the leader unit. However, there is currently no benefit to scaling this charm.
+
 ```bash
-juju run manual-tls-certificates/leader get-outstanding-certificate-requests relation-id=<id>
+juju run manual-tls-certificates/leader get-outstanding-certificate-requests relation-id=${relation_id}
```

Comment:
I would revert this change, to keep the consistency with line 31. The previous version also highlights better when viewed on GitHub.

Reply:
I really despise this format (and "highlights better" is subjective, since it is parsing it incorrectly :laughing:) -- but I suppose that's a discussion for another time.

Removed in https://github.com/canonical/manual-tls-certificates-operator/pull/81/commits/9c3a4d8ec796bb06c3b0de64fc22304ebf0113c7